rosidl_generator_py/resource/_msg.py.template

```diff
@@ -98,7 +102,7 @@ class @(spec.base_type.type)(metaclass=Metaclass):
         self.@(field.name) = kwargs.get(
             '@(field.name)', @(spec.base_type.type).@(field.name.upper())__DEFAULT)
 @[    else]@
-@[      if not field.type.is_primitive_type()]@
+@[      if not field.type.is_primitive_type() and (not field.type.is_array or field.type.array_size)]@
```

Comment:
Could you detail this condition? I you receive a bounded array you import the message module but if it's an unbounded array you don't want to import it ?